pkg/flaps/flaps.go

```diff
@@ -19,47 +19,47 @@ import (
 )
 
 type Client struct {
+	app       *api.App
 	peerIP    string
 	authToken string
 }
 
-func New(ctx context.Context, orgSlug string) (*Client, error) {
+func New(ctx context.Context, app *api.App) (*Client, error) {
 	client := client.FromContext(ctx).API()
 	agentclient, err := agent.Establish(ctx, client)
 	if err != nil {
 		return nil, fmt.Errorf("error establishing agent: %w", err)
 	}
 
-	dialer, err := agentclient.Dialer(ctx, orgSlug)
+	dialer, err := agentclient.Dialer(ctx, app.Organization.Slug)
 	if err != nil {
-		return nil, fmt.Errorf("ssh: can't build tunnel for %s: %s", orgSlug, err)
+		return nil, fmt.Errorf("ssh: can't build tunnel for %s: %s", app.Organization.Slug, err)
 	}
 
 	return &Client{
+		app:       app,
 		peerIP:    resolvePeerIP(dialer.State().Peer.Peerip),
 		authToken: flyctl.GetAPIToken(),
 	}, nil
 }
 
 func (f *Client) Launch(ctx context.Context, builder api.LaunchMachineInput) ([]byte, error) {
-	targetEndpoint := fmt.Sprintf("http://[%s]:4280/v1/machines", f.peerIP)
-
 	body, err := json.Marshal(builder)
 	if err != nil {
 		return nil, err
 	}
 
-	return f.sendRequest(ctx, nil, http.MethodPost, targetEndpoint, body)
+	return f.sendRequest(ctx, nil, http.MethodPost, "", body)
 }
 
 func (f *Client) Stop(ctx context.Context, machine *api.Machine) ([]byte, error) {
-	stopEndpoint := fmt.Sprintf("/v1/machines/%s/stop", machine.ID)
+	stopEndpoint := fmt.Sprintf("%s/stop", machine.ID)
```

Comment:
If I'm following the code in `sendRequest`, we need to prefix all endpoints targeting a specific machine with a `/`.

```suggestion
	stopEndpoint := fmt.Sprintf("/%s/stop", machine.ID)
```